student_class.rb

```diff
@@ -0,0 +1,12 @@
+require './Person'
+
+class Student < Person
+  def initialize(age, name, parent_permission, classroom)
+    super(age, name, parent_permission)
+    @classroom = classroom
```

Comment:
- [ ] in order for this initialize method to work please make sure to set the default values for age and parent_permission as well.